localstack/utils/persistence.py

```diff
@@ -256,4 +257,5 @@ def get_file_path(api, create=True):
                 os.utime(file_path, None)
         if os.path.exists(file_path):
             API_FILE_PATHS[api] = file_path
+        chmod_r(file_path, 0o777)
```

Comment:
I think we can remove this `chmod_r` from here now (shouldn't be required anymore).

Reply:
Yes, good catch